app/controllers/LdapController.java

```diff
@@ -100,8 +101,9 @@ public Result index() {
 
         final List<RoleResponse> roles = Lists.newArrayList(rolesService.loadAll());
         final Set<String> selectedAdditionalGroups = ldapSettings != null && ldapSettings.getAdditionalDefaultGroups() != null ? ldapSettings.getAdditionalDefaultGroups() : Collections.<String>emptySet();
+        final Map<String, String> groupMapping = ldapSettings == null ? new HashMap<String, String>() : ldapSettings.getGroupMapping();
```

Comment:
Minor nit-pick: Instead of creating a new empty hash map, use `Collections.emptyMap()`.